### ISSUE TITLE
Documentation of available parametters T()

### ISSUE THREAD

**Issue by dizzyn** [mohayonao/timbre.js#59]
Hi, I have to say that i like the library and it's really useful.

Can I ask if there is any documentation.

I would appriciate just a simple list of all possible options. In example I see [here](https://mohayonao.github.io/timbre.js/GettingStarted.html) a demo:

`T("perc",` {r:500}, sine1, sine2).on("ended", function() {

But I don't know what means 'perc' as same I don't know what else can be called instead of 'sin', 'saw', 'perc', 'pulse'.

I'm not skilled in synths and electronic sound processing in general and I understand that you have just  implemented something that already exists. Maybe that is my trouble.

Best regards
Tom

**Comment by wilkerlucio** [comment]
I agree with @dizzyn, the library has super useful features, but I'm having a hard time figuring out how to use it, more documentation would be greatly appreciated.
